generator/generator.go

```diff
@@ -591,7 +625,300 @@ func (pg *Generator) GeneratePSPWithName(
 	return psp
 }
 
-func (pg *Generator) fromPodObj(metadata types.Metadata, spec corev1.PodSpec) (string, error) {
+func (pg *Generator) GenerateOPAPodWithName(
```

Comment:
please put some comments in the beginning of the function since this is a public function :)